jetty-core/jetty-deploy/src/test/java/org/eclipse/jetty/deploy/providers/ContextProviderRuntimeUpdatesTest.java

```diff
@@ -80,7 +80,9 @@ public void startJetty() throws Exception
     {
         jetty.addConfiguration("jetty.xml");
         jetty.addConfiguration("jetty-http.xml");
-        jetty.addConfiguration("jetty-deploymgr-contexts.xml");
+        jetty.addConfiguration(MavenPaths.projectBase().resolve("src/main/config/etc/jetty-deployment-manager.xml"));
+        jetty.addConfiguration(MavenPaths.projectBase().resolve("src/main/config/etc/jetty-deploy.xml"));
```

Comment:
These should be there for the test to function.
But instead of doing the 12.0.x technique of duplicating these into `src/test/resources` and having to keep them in sync, it just made more sense to reference the actual XML files directly instead.